docs/configuration/pepr-policies.md

```diff
@@ -0,0 +1,48 @@
+|[Restrict Proc Mount](https://kyverno.io/policies/pod-security/baseline/disallow-proc-mount/disallow-proc-mount/) | [`RestrictProcMount`](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/security.ts#L169-L198) | Subject: **Pod** <br> Severity: **high** <br><br> The default /proc masks are set up to reduce the attack surface. This policy ensures nothing but the specified procMount can be used. By default only "Default" is allowed. |
+|[Restrict Seccomp](https://kyverno.io/policies/pod-security/baseline/restrict-seccomp/restrict-seccomp/) | [`RestrictSeccomp`](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/security.ts#L200-L242) | Subject: **Pod** <br> Severity: **high** <br><br> The SecComp profile should not be explicitly set to Unconfined. This policy, requiring Kubernetes v1.19 or later, ensures that the `seccompProfile.Type` is undefined or restricted to the values in the allowed list. By default, this is `RuntimeDefault` or `Localhost`. |
+|[Restrict SELinux Type](https://kyverno.io/policies/pod-security/baseline/disallow-selinux/disallow-selinux/) | [`RestrictSELinuxType`](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/security.ts#L287-L327) | Subject: **Pod** <br> Severity: **high** <br><br> SELinux options can be used to escalate privileges. This policy ensures that the `seLinuxOptions` type field is undefined or restricted to the allowed list. |
+|[Restrict Volume Types](https://kyverno.io/policies/pod-security/restricted/restrict-volume-types/restrict-volume-types/) | [`RestrictVolumeTypes`](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/storage.ts#L7-L52) | Subject: **Pod** <br> Severity: **medium** <br><br> Volume types, beyond the core set, should be restricted to limit exposure to potential vulnerabilities in Container Storage Interface (CSI) drivers. In addition, HostPath volumes should not be. |
+
+
+### Pepr Policy Mutations
+
+{{% alert-note %}}
+See [Pepr Policy Exemptions](http://localhost:1313/docs/pepr-policies/#pepr-policy-exemptions) for exempting mutations.
+{{% /alert-note %}}
+
+| Pepr Mutation🔗 | Mutated Fields | Mutation Logic |
+| --------------- | -------------- | -------------- |
+| [Disallow Privilege Escalation](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/security.ts#L14-L75) | `securityContext.allowPrivilegeEscalation` | Ensures `allowPrivilegeEscalation` is either undefined or set to `false`. Conditions: the field is undefined, the container is not privileged, and `CAP_SYS_ADMIN` is not added. |
+| [Disallow Privileged Containers](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/security.ts#L14-L75) | `securityContext.privileged` | Ensures that the container does not run in privileged mode by setting `securityContext.privileged` to `false`. |
```

Comment:
Yeah, this one shares the same mutation. Kinda on the fence if it should be documented in some way since, in case someone is trying to find the specific use case for privileged containers vs privileged escalation.

Reply:
Offline discussion, decided to remove the Disallow Privileged Containers infavor of combinging into the Disallow Privileged Escalation